pages/09.contacts/01.manage-contacts/02.contact-monitoring/docs.en.md

```diff
@@ -76,14 +80,16 @@ You can attach custom parameters or overwrite the automatically generated parame
 ```
 This code will send all the automatic data to Mautic and adds also email and firstname. The values of those fields must be generated by your system. 
 
-Beginning in Mautic 2.13, the tracking code also supports company fields. Mautic can assign a company to your tracked contact based on company name. Then you have to add **company** or **companyname** parameter to tracking code along with other companies fields ( companyemail, companyaddress1, companyaddress2, companyphone, companycity, companystate, companyzipcode, companycountry, companywebsite, companynumber_of_employees, companyfax, companyannual_revenue, companyindustry, companyindustry, companydescription...):
+The tracking code also supports company fields. Mautic can assign a company to your tracked contact based on company name. Then you have to add **company** or **companyname** parameter to tracking code along with other companies fields ( companyemail, companyaddress1, companyaddress2, companyphone, companycity, companystate, companyzipcode, companycountry, companywebsite, companynumber_of_employees, companyfax, companyannual_revenue, companyindustry, companyindustry, companydescription...):
+
+Contact tags and UTM codes can also be used. 
 
 ```
-    mt('send', 'pageview', {email: 'my@example.com', firstname: 'John', company: 'Mautic', companyemail: 'mautic@example.com', companydescription: 'description of company', companywebsite: 'https://example.com'});
+    mt('send', 'pageview', {email: 'my@example.com', firstname: 'John', company: 'Mautic', companyemail: 'mautic@example.com', companydescription: 'description of company', companywebsite: 'https://example.com', tags: 'addThisTag,-removeThisTag', utm_campaign: 'Some Campaign'});
 
 ```
 
-#### Load Event
+#### Load event
 
 As the JS tracking request is loaded asynchronously, you can ask JS to call a function when a request is loaded. To do that, define a *onload* function in options like this:
```

Comment:
```suggestion
As the JS tracking request is loaded asynchronously, you can ask JS to call a function when a request is loaded. To do that, define a *onload* function in options like this:
```

Reply:
There doesn't seem to be a change in this?

Reply:
#### Load Event
#### Load event

It was this - but somehow the comment went wonky! Capitalising the E.